src/prif.F90

```diff
@@ -99,13 +100,29 @@ module prif
     type(team_data), pointer :: info => null()
   end type
 
+  abstract interface
+    subroutine prif_stop_callback_interface( &
+          is_error_stop, quiet, stop_code_int, stop_code_char)
+      import :: c_bool, c_int
```

Comment:
If we want to add it to the PRIF spec, it could instead be `use iso_c_binding, only: c_bool, c_int`. We haven't really worried about where the C binding symbols have come from anywhere else in the PRIF spec.

Reply:
I wasn't arguing strongly for inclusion in the spec declaration, I guess I was just surprised to see it here since these were already `use`d at the containing module level and I assumed that would be sufficient.

I'm fine with leaving the spec as-is